modules/ROOT/pages/logging/query-log-analyzer.adoc

```diff
@@ -0,0 +1,101 @@
+[[aura-monitoring]]
+= Query log analyzer
+
+label:AuraDB-Professional[]
+label:AuraDB-Enterprise[]
+
+Query log analyzer is a feature that provides a UI to review the queries executed on an Aura instance.
+
+You can access query logs from *Logs* in the left-hand navigation or from query-related metrics in the *Database* tab in *Metrics*.
+Use the more menu (*...*) on the metric to navigate to *Explore query logs*.
+
+To toggle between instances, use the dropdown menu on the top left.
+
+Query log analyzer is split up in three parts:
+
+* *Query timeline* - Timeline showing metrics for number of queries, failed queries and query latency.
+* *Summary table* - An aggregated view of query logs, giving a high level overview over the selected time period.
+* *Details table* - A detailed view showing individual query executions in the selected time period.
+
+To fetch logs, first select a time range in the Query timeline.
+With a time selection done, press the *Fetch logs* button.
+You may optionally select any filters or search text if required, then press *Fetch*.
+
+A summary of query executions is returned, showing aggregations per query.
+To see the individual query executions, click the right arrow at the end of the line to show details for that query.
+The details pane shows individual executions.
+
+== Timeline interactions
+
+When viewing the query timeline, you can select from the following time intervals:
+
+* 30 minutes
+* Last hour
+* Last 2 hours
+* Last 6 hours
+* Last 24 hours
+* Last 3 days
+* Last week
+
+The query timeline can be collapsed by clicking on the header.
+
+=== Zoom
+
+To zoom in to a narrower time interval, select and drag inside the timeline to select your desired time interval.
+The data in the timeline will automatically update to match the increased resolution.
+To update the table, click the *Fetch logs* button.
+
+To reset zoom, double-click anywhere inside the timeline.
+
+=== Toggle data series
+
+To hide or show individual data series, select the corresponding data series in the legend below the timeline.
+
+
+== Fetch logs
+
+The *Fetch logs* button will open up a dialog where you can add filters and search before fetching the logs.
+The Query timeline determines the current time selection, which can be changed by closing the dialog and modifing the timeline.
+To fetch the logs after selection of filters and search is done, click the *Go* button.
+
+=== Filters
+
+To filter, click the filter button.
+This will load the available filters over the selected time period.
+Filters are available for the following fields:
+
+* Status
+* User
+* Driver
+* Application
+* Initiation type
+
+=== Search
+
+To search, click the search button.
+Search can be specified for the *Query text* and the *Error text*.
+The fields are case insensitive and allows you to find specific queries or error that are interesting.
```

Comment:
```suggestion
The fields are case insensitive and allow you to find specific queries or errors of interest.
```
I know this sentence was lifted from the existing document, but it could be improved.

Reply:
Rewrote: 
Use the search button to search for specific queries or errors.
Search can be specified for the *Query text* and the *Error text*.
The fields are case insensitive.